ansible/templates/etc/systemd/system/oauth2-proxy-admin.service.j2

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=The Koala authentication daemon, admin-only version
+Wants=koala.service
+After=koala.service
+OnFailure=failure-notificator@%n.service
+
+[Service]
+Restart=always
+ExecStart=/usr/local/bin/oauth2_proxy -config /etc/oauth2_proxy/config-admin.cfg
```

Comment:
This doesn't match with where the config file is copied to and the name of the template file

```suggestion
ExecStart=/usr/local/bin/oauth2_proxy -config /etc/oauth2_proxy/admin-config.cfg
```